config/module.config.php

```diff
@@ -31,6 +31,12 @@
                 'hydrator_dir'       => 'data/DoctrineMongoODMModule/Hydrator',
                 'hydrator_namespace' => 'DoctrineMongoODMModule\Hydrator',
 
+                'generate_persistent_collections' => Configuration::AUTOGENERATE_ALWAYS,
```

Comment:
Out of scope for this PR, but is it common for other Zend modules to use `AUTOGENERATE_ALWAYS` for proxies and hydrators? ODM bundle for instance does [the opposite](https://github.com/doctrine/DoctrineMongoDBBundle/blob/ee1cdebc02d4ed49e90e9ad15737b1be03ac3eee/DependencyInjection/Configuration.php#L51) relying on generating proxies/hydrators/collections during cache warmup. Maybe it's something that could be improved for 1.0.0?